### ISSUE TITLE
Remove macOS menu bar icon

### ISSUE THREAD

**Issue by rosmur** [ollama/ollama#7889]
Can we please get the ability to turn off the macOS menu bar icon?

**Comment by alex-pex** [comment]
I was looking for the method to hide it, I'm surprised there's none yet

**Comment by anotherbazeinthewall** [comment]
+1!

**Comment by indeyets** [comment]
to add some context on why it is needed: menubar estate is limited (especially on laptop displays) and is often overcrowded by indicators. any non-essential icon means that other icons wouldn't fit and will be hidden.

on the contrast, dock-estate is almost infinite

**Comment by hyuri** [comment]
Correct.

Also, menu bar icons catch attention, because they signal state. Since there's nothing the Ollama icon needs to tell us, and there's no need to interact with it, there's no need for an icon there, it only distracts.

**Comment by rosmur** [comment]
@ParthSareen @jmorganca for attention. Any updates here -  looks like this has a lot of traction from the community and has been open for 4 months

**Comment by Fynnwind** [comment]
I neither wanted the macOS menu bar icon nor the dock icon, so I installed the ollama formular not the cask with brew and ran `brew services start ollama` to ensure ollama is active upon login.
I don't see the point of the app in its current state, I'll install it if it becomes a proper model hub (including a feature to hide the menu bar icon ;D).

**Comment by rosmur** [comment]
you install with just `brew install ollama`

**Comment by vincvincvinc** [comment]
I agree with Hyuri and Indeyets, should be normal option since it has no use and blocks other things ...

**Comment by hermeticvm** [comment]
How does one get rid of the Dock icon though? It used to work just living in the tray/menu bar but ever since Ollama shipped their own chat UI it's polluting my Dock.

**Comment by rosmur** [comment]
Yes, the GUI was the last straw for me. Its the push I needed to uninstall and go to llama.cpp+llama-swap. Even Docker Model Runner that ships with Docker Desktop seems to be great

**Comment by gotofritz** [comment]
Another +1, this has been open for months and should be a trivial change

**Comment by rosmur** [comment]
yes and the fact that they havent done it tells us everything about their intent. Its a blatant dark pattern.

I've gotten setup on llama-swap+llama.cpp and have uninstalled ollama, so Im closing this thread as abandoned

**Comment by sordahl** [comment]
you can just cmd+drag the icon on to the desktop to remove Ollama icon from menu bar?

**Comment by gotofritz** [comment]
Have you tried it, @sordahl?

**Comment by sidnevrekar** [comment]
U can use command and drag it off. It should work it worked for me.

**Comment by sidnevrekar** [comment]
Or use something lke alfred or ice to control your dock.

**Comment by gotofritz** [comment]
No it it doesn't work. And I don't want to install another piece of software to handle something that ollama should do better.
Finally, we are not talking about the dock but the menu bar. Thanks for your contribution @sidnevrekar , I will unsubscribe now as I am no longer using ollama and the issue is closed anyway.

**Comment by sordahl** [comment]
@rosmur @gotofritz Yes, I have, works on Tahoe – Had the exact same frustration as you, when dragging, you have to drag it out a bit and wait a second or two.

Also, In Tahoe you can go to settings > menu bar > "Allow in the menu bar" > 	disable check on Ollama and it's gone
